include/wx/osx/cocoa/private/textimpl.h

```diff
@@ -129,7 +129,7 @@ class wxNSTextViewControl : public wxNSTextBase
 
 #if wxUSE_SPELLCHECK
     virtual void CheckSpelling(const wxTextProofOptions& options) wxOVERRIDE;
-    virtual bool IsSpellingCheckEnabled() const wxOVERRIDE;
+    virtual wxTextProofOptions GetCheckingOptions() const wxOVERRIDE;
```

Comment:
Can't this be `GetProofCheckOptions()` too or is it for backwards compatibility with an existing function?

Reply:
No, this is a private header, so there are no compatibility concerns here, but I find calling these private functions using the same names as the public ones confusing, so I wanted to use a different name. And this one is consistent with what Cocoa uses (the properties enabling different checks are called `xxxChecking`), so I'd rather leave it as is.

Again, this is invisible to wx users anyhow and can always be changed later if really needed.